task.sql

```diff
@@ -38,3 +38,16 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+--changeset fredy.zoba:1 labels:0.0.2
+CREATE TABLE Users (
+    ID INT PRIMARY KEY AUTO_INCREMENT,
+    FirstName VARCHAR(50),
+    LastName VARCHAR(50),
+    Email VARCHAR(50)
+);
+--rollback DROP TABLE Users;
+
+--changeset fredy.zoba:2 labels:0.0.3
+CREATE INDEX index_email ON Users (Email);
```

Comment:
The index creation is correctly defined. Ensure that the `Email` column in the `Users` table is intended to be indexed for performance reasons, such as frequent search operations.